### DISCUSSION TITLE
Missed the Orientation session on 29 oct, 2022

### DISCUSSION THREAD

**Discussion by faisal1025** [github/india#549] [Students]
Actually, I have missed the orientation session which was going to be held on 29 oct, 2022 but I was unable to attend this session although I have filled the profile requirements to unlock self-learning and Industry Session and also the links section but still my next sections are unlock what to do next please help me out.

**Comment by RhugwedaShedge** [comment]
At the bottom of the orientation section, you will find the recording of the session.
